watertap/costing/watertap_costing_package.py

```diff
@@ -117,6 +117,13 @@ def build_global_params(self):
             units=self.base_currency / pyo.units.kWh,
         )
 
+        self.electrical_carbon_intensity = pyo.Param(
+            mutable=True,
+            initialize=0.1,

```

Comment:
Is this value notional? It seems a bit optimistic, at least according to what I can find googling around.

Reply:
Thanks for the catch! The recent commit updates this value based on the [2019 IEA Global Energy & CO2 Status Report 2019](https://www.iea.org/reports/global-energy-co2-status-report-2019): ~0.475 kgCO2eq/kWh.